### DISCUSSION TITLE
Dobbelt oprettelse af vagtdage

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#41] [Bugs] [answered]
Man kan på en eller anden måde oprette en vagtdag to gange. F.eks. her: 

![image](https://github.com/xooom-community/product/assets/167326849/fd728df2-d333-4bae-8609-f2f5927299ad)

**Comment by mikkelsl** [comment, marked as answer]
Ja, dagene er egentlig tænkt som eventdatoer.
Det kan f.eks. bruges, hvis samme event foregår 2 gange på en dag.

Jeg tænker ikke, der sker noget ved det her, selvom det måske er unødvendigt.

**Reply by MartinQwist** [reply]
OK. Vi gør ikke noget nu. Så længe det ikke forstyrrer visningen, så skaber det højst lidt tvivl.
